modules/fbs-core/web/src/app/page-components/sql-playground/sql-input-tabs/sql-input-tabs.component.ts

```diff
@@ -130,7 +130,9 @@ export class SqlInputTabsComponent
 
   addTab() {
     this.tabs.push({ name: this.fileName, content: "" });
-    this.activeTabId.setValue(this.tabs.length - 1);
+    setTimeout(() => {
+      this.activeTabId.setValue(this.tabs.length - 1);
+    }, 15);
```

Comment:
This solution works, but is more of a workaround. The error lies in the fact that when the plus button is clicked, the click of the tab is also executed and thus the active tab is set back to the first one. Therefore, we can call an `event.stopPropagation()` at the beginning of the Methode `addTab` and thus prevent this.

To call `event.stopPropagation()` the event must be passed to the method `addTab`